src/index.html

```diff
@@ -0,0 +1,148 @@
+
+            <div class="products__info">
+              <h2 class="products__name">Speed Bike Emigrantes</h2>
+
+              <p class="products__price">R$ 5640,00</p>
+
+              <button class="button-add">Adicionar ao Carrinho</button>
+            </div>
+          </li>
+
+          <li class="products__list">
+            <img class="products__img" src="assets/images/products/chinatown-classic-bike.png" alt="Chinatown Classic Bike" />
+
+            <div class="products__info">
+              <h2 class="products__name">Chinatown Classic Bike</h2>
+
+              <p class="products__price">R$ 799,90</p>
+
+              <button class="button-add">Adicionar ao Carrinho</button>
+            </div>
+          </li>
+
+          <li class="products__list">
+            <img class="products__img" src="assets/images/products/buscape-tandem-bike.png" alt="Tandem Bike" />
+
+            <div class="products__info">
+              <h2 class="products__name">Tandem Bike</h2>
+
+              <p class="products__price">R$ 2600,00</p>
+
+             <button class="button-add">Adicionar ao Carrinho</button>
+            </div>
+          </li>
+
+          <li class="products__list">
+            <img class="products__img" src="assets/images/products/crosses-bmx.png" alt="Cross BMX" />
+
+            <div class="products__info">
+              <h2 class="products__name">Cross BMX</h2>
+
+              <p class="products__price">R$ 1950,00</p>
+
+              <button class="button-add">Adicionar ao Carrinho</button>
+            </div>
+          </li>
+
+          <li class="products__list">
+            <img class="products__img" src="assets/images/products/tyson-gloves.png" alt="Luvas Tyson" />
+
+            <div class="products__info">
+              <h2 class="products__name">Luvas Tyson</h2>
+
+              <p class="products__price">R$ 189,00</p>
+
+              <button class="button-add">Adicionar ao Carrinho</button>
+            </div>
+          </li>
+
+          <li class="products__list">
+            <img class="products__img" src="assets/images/products/squeezer-squirtle.png" alt="Squeezer Squirtle" />
+
+            <div class="products__info">
+              <h2 class="products__name">Squeezer Squirtle</h2>
+
+              <p class="products__price">R$ 30,00</p>
+
+              <button class="button-add">Adicionar ao Carrinho</button>
+            </div>
+          </li>
+
+          <li class="products__list">
+            <img class="products__img" src="assets/images/products/dragon-born-helmet.png" alt="Capacete Dragon Born" />
+
+            <div class="products__info">
+              <h2 class="products__name">Capacete Dragon Born</h2>
+
+              <p class="products__price">R$ 650,90</p>
+
+              <button class="button-add">Adicionar ao Carrinho</button>
+            </div>
+          </li>
+        </ul>
+      </section>
+
+    </main>
+
+  </body>
+
+</html>
```

Comment:
Remover essas linhas extras:

```suggestion
      </section>
    </main>
  </body>
</html>
```